src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-detail.component.ts

```diff
@@ -73,6 +93,26 @@ export class FileUploadExerciseDetailComponent implements OnInit, OnDestroy {
         });
     }
 
+    getExerciseDetailSections() {
```

Comment:
```suggestion
    private getExerciseDetailSections() {
```

This can be private.

Reply:
To keep it easier testable (and Artemis in General has no strict differentiation between private and public methods) I would like to keep it public